frontend/components/ProductSlider/index.jsx

```diff
@@ -7,23 +7,36 @@ import withRecommendations from '../../connectors/withRecommendations';
  * General Product Slider component.
  * @param {string} type Type.
  * @param {string} id Id.
+ * @param {Object} settings widget settings.
  * @returns {JSX}
  */
-const ProductSlider = ({ type, id }) => {
+const ProductSlider = ({
+  type, id, limit, settings,
+}) => {
   const ConnectedSlider = withRecommendations(
     Slider,
     {
       type,
       id,
+      limit,
+      settings,
     }
   );
 
   return <ConnectedSlider />;
 };
 
 ProductSlider.propTypes = {
-  id: PropTypes.string.isRequired,
   type: PropTypes.string.isRequired,
+  id: PropTypes.string,
+  limit: PropTypes.number,
+  settings: PropTypes.shape(),
```

Comment:
Please add shape prop types so it's obvious what's wrong on live.

Reply:
done. but for live it will not help since we don't validate proptypes there afaik

Reply:
I like it as a description for human what to expect.